mergify_engine/tests/functional/actions/test_post_check.py

```diff
@@ -116,6 +117,7 @@ def test_checks_custom(self):
 class TestPostCheckActionNoSub(base.FunctionalTestBase):
     def test_checks_feature_disabled(self):
         self.subscription = subscription.Subscription(
+            asyncio.run(utils.create_aredis_for_cache(max_idle_time=0)),
```

Comment:
Does that make even sense?

Reply:
It's not really logic but it works because of the aredis implementation.

All tests should be async, but pytest-asyncio does not work for tests using unittest.TestCase :( 
I don't know yet how to fix that. Maybe we need to rewrite the setUp and all helpers with pytest, but that a lot of works (and a lot of tox -e record).

Reply:
I just found python3.8 have introduced unittest.IsolatedAsyncioTestCase, I will make a separate PR for cleaning this.